web/gui/dashboard.js

```diff
@@ -2344,32 +2374,40 @@ var NETDATA = window.NETDATA || {};
                     if (min > max) max = min;
 
                     // find the smallest scale that provides integers
-                    for (x in this.scalableUnits[units]) {
-                        if (this.scalableUnits[units].hasOwnProperty(x)) {
-                            var m = this.scalableUnits[units][x];
-                            if (m <= max && m > tdivider) {
-                                tunits = x;
-                                tdivider = m;
-                            }
+                    // for (x in this.scalableUnits[units]) {
+                    //     if (this.scalableUnits[units].hasOwnProperty(x)) {
+                    //         let m = this.scalableUnits[units][x];
+                    //         if (m <= max && m > tdivider) {
+                    //             tunits = x;
+                    //             tdivider = m;
+                    //         }
+                    //     }
+                    // }
+                    const sunit = this.scalableUnits[units]
```

Comment:
missing semicolon.

Reply:
Will add it.

(FWIW, semicolons are not really required in JS, unlike C, in my own code I don't add semicolons, but here let's keep them because it's a more standard coding convention)